src/govuk/components/pagination/pagination.yaml

```diff
@@ -0,0 +1,169 @@
+params:
+- name: previous
+  type: object
+  required: false
+  description: A link to the previous page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The previous page link text. Defaults to 'Previous page', where 'page' is visually hidden.
+  - name: label
+    type: string
+    required: false
+    description: Optional label that goes underneath the previous page link providing further context for where it goes.
+  - name: href
+    type: string
+    required: true
+    description: The previous page URL.
+- name: next
+  type: object
+  required: false
+  description: A link to the next page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The next page link text. Defaults to 'Next page', where 'page' is visually hidden.
+  - name: label
+    type: string
+    required: false
+    description: Optional label that goes underneath the next page link providing further context for where it goes.
+  - name: href
+    type: string
+    required: true
+    description: The next page URL.
+- name: type
+  type: string
+  required: false
+  description: The type of navigation. Goes in the aria label for the encapsulating `nav` element. Defaults to "results".
+- name: items
+  type: array
+  required: false
+  description: Array of link objects.
+  params:
+  - name: number
+    type: number
```

Comment:
Is this really a string? We don't treat it as a number in any meaningful sense, and we just output it as text.